pandas/tests/indexes/multi/test_duplicates.py

```diff
@@ -332,7 +332,7 @@ def test_multi_drop_duplicates_pos_args_deprecation():
     idx = MultiIndex.from_arrays([[1, 2, 3, 1], [1, 2, 3, 1]])
     msg = (
         "In a future version of pandas all arguments of "
-        "MultiIndex.drop_duplicates will be keyword-only"
+        "Index.drop_duplicates will be keyword-only"
```

Comment:
Okay given that our next release is 2.0 which we can remove this test entirely (otherwise the previous error message was a little better)

Reply:
maybe we can adjust the deprecation decorator to always print the current class name

Reply:
That would be a good enhancement/followup. Mind opening an issue about that?

Reply:
Opened #48650